web/src/beta/components/LayerSectionField/item.tsx

```diff
@@ -0,0 +1,92 @@
+import { useCallback } from "react";
+
+import Icon from "@reearth/beta/components/Icon";
+import Text from "@reearth/beta/components/Text";
+import { SwitchField } from "@reearth/beta/hooks/useManageSwitchState/hooks";
+import { styled, useTheme } from "@reearth/services/theme";
+
+export type LayerSectionItemProps = {
+  label: string;
+  visible?: boolean;
+};
+
+type Props = {
+  layer: SwitchField<LayerSectionItemProps>;
+  onActive?: (id: string) => void;
+  onAction?: (id: string) => void;
+  onVisible?: (id: string) => void;
+};
+
+const LayerSectionItem: React.FC<Props> = ({ layer, onActive, onAction, onVisible }) => {
+  const theme = useTheme();
+  const handleActive = useCallback(() => {
+    onActive?.(layer.id);
+  }, [layer.id, onActive]);
+  const handleVisible = useCallback(() => {
+    onVisible?.(layer.id);
+  }, [layer.id, onVisible]);
+  const handleAction = useCallback(() => {
+    onAction?.(layer.id);
+  }, [layer.id, onAction]);
```

Comment:
I think this needs stopPropagation(). Right now if I click the action button this AND handleActive are executed.